draw_extras.lua

```diff
@@ -1,55 +1,49 @@
+local math 		= math
+local surface 	= surface
+
 -- Draws a filled circle
```

Comment:
smh, misleading ~~function name~~ comment

Reply:
Whoops didn't see that, thanks